docs/3.0/develop/blocks.mdx

```diff
@@ -177,13 +164,16 @@ To load the `S3Bucket` block from above, run the following:
 ```python
 from prefect_aws.s3 import S3Bucket
 
-s3_block = Block.load("S3Bucket/my_s3_bucket_block")
+s3_block = Block.load("s3-bucket/my_s3_bucket_block")
 print(s3_bucket.bucket_name) # data-bucket
 ```
 
-### Delete blocks
+<Warning>
+When using the generic `Block.load` interface for block documents, the Python class that created the type _must_ be importable.
+For this reason, we highly recommend loading block documents by first importing the block class and then calling its `load` method with the relevant document name.
+</Warning>
```

Comment:
This warning is a duplicate. Not sure if that's intentional.

Reply:
Yea that was intentional for now, just because I know this is a common trap people fall into - I considered making it a snippet but since it's in the same file I figured that was unnecessary (for now at least)

Reply:
i think there are actually 3 instances of this warning on this page, which all boil down to us not storing the actual class on the server so i think removing at least 1 makes sense

Reply:
Yea sounds good - I removed the one that wasn't immediately preceded by a `load` example 👍